api/src/controllers/record-controller.js

```diff
@@ -169,6 +169,13 @@ exports.protectedPost = async function (args, res, next) {
 
     let response = await Promise.all(promises);
 
+    // Audit the POST action.
+    // If multiple observables are triggered, response will have multiple objects
+    // only add the metaId for single objects.
+    let meta = response && response[0] && response[0][0] ? response[0][0] : null;
+    let metaID = meta ? meta.object[0]._id : null;
```

Comment:
Does this require null checking similar to the line above (175)?

Reply:
Yes. Check has been added to ensure object is there